x-pack/plugin/security/src/test/java/org/elasticsearch/integration/IndexPrivilegeTests.java

```diff
@@ -419,6 +432,22 @@ private void assertUserExecutes(String user, String action, String index, boolea
                 }
                 break;
 
+            case "maintenance" :
+                if (userIsAllowed) {
+                    assertUserIsDenied(user, "crud", index);
```

Comment:
I think this this is what we want.
A user beling allowed `maintenance` should not imply anything about whether they are/aren't allowed `crud`.

Reply:
@tvernum the first and second statement made me a bit confuse. but it looks like the explanation has precedence. so I will remove them.